text/0000-fabric-private-chaincode-lite.md

```diff
@@ -41,17 +42,37 @@ Such data includes secret cryptographic keys, which the chaincode uses to secure
     and also still mention all possible use-cases. 
     The distinction is done later in the architecture/design section
 -->
+
+***TODO*** document is missing a bit in the intro section about:
+- clear statement what is the purpose of the document
+- link to full FPC
+- explain that we begin with FPC lite on our journey to FULL FPC security features
+- differentiate between FPC Lite/Full Security properties and the feature to support vanilla Fabric- clarify
+- introduce MVP a bit earlier
+
 ## Enable new use-cases with strong privacy requirements
 FPC is motivated by the many use cases in which it is desirable to embody an application in a Blockchain architecture, but where in addition to the *existing integrity assurances*, the application *also requires privacy*. This may include private voting, sealed bid auctions, operations on sensitive data such as regulated medical or genomic data, and supply chain operations requiring contract secrecy. With Fabric's current privacy mechanisms, these use cases are not possible as they still require the endorsement nodes to be fully trusted.
 For example, the concept of channels and Private Data allows to restrict chaincode data sharing only within a group of authorized participants, still when the chaincode processes the data it is exposed to the endorsing peer in clear. In the example of a voting system, where a government may run an endorsing peer it is clear that this is not ideal.
 
 ## Enable performance improvements
 <!-- Below section does not apply to FPC Lite but as we haven't introduced that and this motivation can be address in a later extensions we still mention it here -->
 A second motivation for FPC is its integrity model based on hardware-enabled remote cryptographic attestation; this model can provide similarly high confidence of integrity to the standard Fabric model of integrity through redundancy, but using less computation and communication. With TEE-based endorsement and remote attestation, a new set of endorsement policies are made possible, which can reduce the number of required endorsements and still provide sufficient assurance of integrity for many workloads.
+***TODO*** this contradicts our enclave endorsement validation point; make clear that the actual Chaincode Execution may be expensive;
```

Comment:
I guess we could say that. We might also mention more that FPC Lite only has partially tapped into that potential and more could be done by extension and a slightly tighter integration (into already existing extensions points [with that part probably left unspoken ;-;])